### DISCUSSION TITLE
Please add delete history per item feature

### DISCUSSION THREAD

**Discussion by Arxyma** [monkeytypegame/monkeytype#1182] [Ideas] [answered]
My friend used my account and go to zen mode, but he just randomly slam the keyboard and broke my history. Now i can't see my progress graph.
I want to delete his history but the only way to delete is delete the entire history.

**Comment by Miodec** [comment]
What do you mean you cant see your progress graph

**Comment by unrealapex** [comment]
> My friend used my account and go to zen mode

I suggest not sharing accounts with others. 
>  he just randomly slam the keyboard and broke my history. Now i can't see my progress graph.

Detail a bit more, and if possible add screenshots and videos so we can further inspect and diagnose the problem.

**Comment by Arxyma** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/48849062/113743153-8aa00f00-972d-11eb-9963-fd3a87f337a9.png)
my highest score is 84 wpm, but now my friend's score is the highest. i think it will break the average score, and i just want to delete that score.
sorry i forgot to add the sceenshot.

**Reply by unrealapex** [reply]
> my highest score is 84 wpm, but now my friend's score is the highest.

Your averages have lowered because the statistics your friend contributed to your accounts are outliers to your stats when you use Monkeytype. 

Sadly Monkeytype doesn't have a way to clear account statistics, but there are some solutions:
- Deleting your account, essentially clearing your stats from the database, then creating a new account
- Keep practicing on Monkeytype; your averages will gradually come up to their previous values or higher

**Reply by Miodec** [reply]
You could:
- create a tag and tag the results you don't want to see (by filtering out the tag in advanced filters)
- filter out just the zen mode

In general I don't allow for deleting results because people would end up cherry picking results to make their stats look better than they really are. I could delete some of the results for you in this case, if you contact me on Discord.
